docs/antora.yml

```diff
@@ -25,6 +25,7 @@ asciidoc:
     seqdiag-version: 3.0.0
     structurizr-version: 1.30.1
     svgbob-version: 0.7.0
+    symbolator-version: 1.2.1
```

Comment:
Please update the `tasks/update-versions.js` script to automatically update Symbolator version (based on the version defined in the Dockerfile).

We are already doing this for other diagram libraries so you can copy/paste. Let me know if you need help.

Reply:
:+1: It now also updates the symbolator version used in github actions based on the Dockerfile.